zh/backup-restore-snapshot-perf.md

```diff
@@ -84,19 +84,19 @@ EBS 卷快照恢复阶段包含以下阶段，详细信息，参考[基于 EBS 
 
 | 恢复阶段     | 恢复大致耗时 | 恢复总占比 | 备注                                                                            |
 | :--------: | :---------: | :------: | :----------------------------------------------------------------------------: |
-| 创建集群     | 30s         | 1%      | 包含下载 docker image 和启动 PD 的时间                                                 |
-| 卷恢复      | 20s         | 1%     | 包含启动 BR pod 和卷恢复的时间                                                         |
-| 启动 TiKV   | 2～30 min    | 58%      | TiKV 启动时长受卷性能影响，一般 3 分钟左右，当恢复数据包含事务时，此阶段耗时可能达到 30 分钟 |
-| 数据恢复阶段 | 2～20 min    | 38%       | 包含恢复 raft 共识层数据以及删除 MVCC 数据的时间                                      |
-| 启动 TiDB   | 1 min        | 2%       | 包含下载 docker image 和启动 TiDB 的时间                                           |
+| 创建集群     | 30 s         | 2%      | 包含下载 docker image 和启动 PD 的时间                                             |
+| 卷恢复      | 20 s         | 1%      | 包含启动 BR pod 和卷恢复的时间                                                     |
+| 启动 TiKV   | 10~16 min    | 42%     | 包含 TiKV 启动时，启动 rocksdb，读取所有 region 元信息。                             |
```

Comment:
```suggestion
| 启动 TiKV   | 10～16 min    | 42%     | 包含 TiKV 启动时，启动 RocksDB 及读取所有 Region 元信息的时间。                             |
```

Reply:
fixed in https://github.com/pingcap/docs-tidb-operator/pull/2169/commits/e34683931ef9c4c531dfdc857f45a2597b396a44